docs/api/url_search_params.md

```diff
@@ -0,0 +1,99 @@
+# 🔗 URLSearchParams
+
+`jstz` implementation of an API with utility methods for working with the query string of a URL according to the [URLSearchParams specification](https://url.spec.whatwg.org/#urlsearchparams). It is used for building and manipulating search parameters.
+
+## Example
+
+```typescript
+// Parse query string from URL
+let url = new URL("https://example.com?foo=1&bar=2");
+let params = new UrlSearchParams(url.search);
+
+// Add a new parameter
+params.append("baz", 3);
+
+// Remove parameter
+params.delete("bar");
+
+// Loop through parameters
+for (let [key, value] of params) {
+  // ...
+}
+```
+
+## Constructor
+
+### `UrlSearchParams(values: Array<[Name: string, Value: string]>, url?: URL)`
```

Comment:
```suggestion
### `new UrlSearchParams(init?: [string, string][] | Record<string, string> | string)`
```

In future always refer to the `jstz-types` package for the types + parameter names